lib/SPIRV/SPIRVWriter.cpp

```diff
@@ -2039,6 +2042,28 @@ void LLVMToSPIRV::oclGetMutatedArgumentTypesByBuiltin(
     ChangedType[1] = getSamplerType(F->getParent());
 }
 
+SPIRVValue *
+LLVMToSPIRV::transBuiltinToConstant(const std::string &DemangledName,
+                                    CallInst *CI) {
+  Op OC = getSPIRVFuncOC(DemangledName);
+  if(!isSpecConstantOpCode(OC))
+    return nullptr;
+  Type *Ty = CI->getArgOperand(1)->getType();
+  assert(Ty == CI->getType() && "Type mismatch!");
+  Value *V = CI->getArgOperand(1);
+  uint64_t Val = 0;
+  if(Ty->isIntegerTy())
```

Comment:
```suggestion
  if (Ty->isIntegerTy())
```

The same at line 2057

Reply:
Fixed, thanks.